pkg/tfvars/tfvars.go

```diff
@@ -3,6 +3,9 @@ package tfvars
 type TFVars struct {
 	ReleaseMarker  string `json:"release_marker"`
 	BuildVersion   string `json:"build_version"`
+        StorageServer  string `json:"s3_server"`
```

Comment:
see if we can `omitempty` entry to these variables so that when we don't mention these params, it will use the default ones.